cmd/remove.go

```diff
@@ -53,8 +53,10 @@ func init() {
 				raw.fromTo = common.EFromTo.BlobTrash().String()
 			} else if srcLocationType == common.ELocation.File() {
 				raw.fromTo = common.EFromTo.FileTrash().String()
+			} else if srcLocationType == common.ELocation.BlobFS() {
+				raw.fromTo = common.EFromTo.BlobFSTrash().String()
```

Comment:
(Not sure where to leave this comment). You mentioned that include/exclude will not be supported for ADLS Gen 2 remove, but I don't see any error message that's displayed if they are provided when doing an ADLS Gen 2 remove.